sdk/src/main/scala/com/stratio/sparkta/sdk/ValidatingPropertyMap.scala

```diff
@@ -46,8 +46,8 @@ class ValidatingPropertyMap[K, V](val m: Map[K, V]) {
 
   def getString(key: K, default: Option[String]): Option[String] = {
```

Comment:
There are 20 calls to this method, only one of them has a default value, mostly other has None as second param..., we need to point to the "def getString(key: K): String" these calls